message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -93,37 +93,37 @@
       <entry value="42004" name="MAV_CMD_FIXED_MAG_CAL">
         <description>Magnetometer calibration based on fixed position
         in earth field given by inclination, declination and intensity.</description>
-        <param index="1">MagDeclinationDegrees.</param>
-        <param index="2">MagInclinationDegrees.</param>
-        <param index="3">MagIntensityMilliGauss.</param>
-        <param index="4">YawDegrees.</param>
+        <param index="1" label="Declination" units="deg">MagDeclinationDegrees.</param>
```

Comment:
Now we're getting labels - once GCSs start to use them we can improve the descriptions right?

Reply:
I certainly hope we can. Note though that QGC at least often doesn't use the description, but hand crafts its own.